asreview/webapp/task_manager/task_wrapper.py

```diff
@@ -0,0 +1,25 @@
+import json
+import multiprocessing as mp
+import socket
+
+
+class RunModelProcess(mp.Process):
```

Comment:
Move this to `task_manager.py`? It makes it easier to share the port and host (now the default is hardcoded in 2 places).